brand/admin.py

```diff
@@ -326,7 +326,16 @@ def save_model(self, request, obj, form, change):
         This function is to create object of commentary model when default values are
         provided by user in admin portal and save the data in respective database.
         """
-        super().save_model(request, obj, form, change)
+        # Check if there's already a brand with the same tag
+        if (
+            Brand.objects.filter(tag=obj.tag).exists() and not change
```

Comment:
@Harmeetkaur13 - this is a picky thing, but you probably want to test for `not change` before testing for `.exists()`. The reason is that `.exists()` will have to make a database query whereas change is arelady stored in memory.